proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+
+<lynX> Copyright abolition would not solve this problem. Content goes viral using one specific URL and whoever has that copy makes all the money. Still, the idea of going back to "broadcast" is very good and the way to achieve could be to (a) make it mandatory to distribute websites by multicast pubsub instead of HTTP (see https://secushare.org/pubsub or IPFS for that), (b) make targeted advertising illegal and (c) introduce a mandatory anonymous micropayment system by which *any* article can be bought anonymously for a few cents. See https://taler.net for the most promising payment system in that regard.
+
+As for 1.): Besides directly using their power over corporations to install backdoors into their products the intelligence apparatus pays hackers for so called zero day exploits (Knowledge about flaws in security of products that are not known to the public or the vendor of the product and thus can not be fixed by the vendor). Thus these government organizations that should careabout our security actually make us less secure. The practice of paying for exploits that are not disclosed to the public needs to be stopped immediately. Instead we need to publicly fund security research and could offer bounties to individuals who help to find security vulnerabilities.
+
+<lynX> Yes, and forbid proprietary code in all societally essential systems, therefore giving any programmer on Earth a decent chance to find vulnerabilities by reading *source code* rather than disassembling machine code.
+
+Social media systems are useful and popular, yet they are a risk to our privacy. We need a system where it is safe for us to share whatever we want with our family and friends without the risk to share it with third parties. We need public funding for alternative social media systems which are built not for profit and which are based on decentralized systems and strong cryptography to protect our data from getting into the wrong hands.
+
+<lynX> Oh yeah! We need funding to *create* a truly distributed and private social network. None of the existing softwares can actually replace Facebook *technically*. In most cases they are just yet another server you have to trust. See https://secushare.org
+
+We also need to have the guaranteed right to use pseudonym accounts online where we want to.
+<lynX> Why? And is it enough if it is just a right and not an obligation? For the defense of democracy it probably must be mandatory to anonymise or at least pseudonymise digital activity.
+
+Even with all the measures above, some of the new technologies inevitably lead to situations where we leave ever more digital traces of our life (e.g. the existence of cell-phones requires that the cell-phones log into the next-cell phone tower - thus we inevitably leave a trace of our movement.)
```

Comment:
This can actually be solved technically and should be mandated by law. See https://youbroketheinternet.org/programme for details. - lynx